.gitlab/nersc-gitlab-ci.yml

```diff
@@ -0,0 +1,71 @@
+# Kokkos nightly tests at NERSC machine Perlmutter.
+stages:
+  - test
+  - clean_up
+
+default:
+  tags:
+    - perlmutter-login
```

Comment:
Is this still running on a login node? Meaning we might get false positives from the nightlies timing out due to missing separation between jobs running?

Reply:
Yeah this is still running on a login node. We do not have a runner for compute node so unless I submit a job (in which case the CI setup will look very different), I have to run this on a login node. The advantage is that if I limit the resources (like threads/cores used), most of the functionality tests do complete running. Since we also have a GPU on the login node, those tests also complete.